Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ModelChoiceList class is deprecated since SF 2.7'.
```

Comment:
Is this message ok `The XXX class is deprecated since 3.21 and will be removed when bumping requirement to SF 2.7+`? If it's not could you please help me writing this deprecation message?